Makefile

```diff
@@ -122,8 +122,8 @@ acceptance:
 check:
 	@echo "checking for tabs in shell scripts"
 	@! git grep -F '	' -- '*.sh'
-	@echo "checking for \"path\" imports"
-	@! git grep -F '"path"' -- '*.go'
+	@echo "checking for forbidden imports"
+	@! go list -f '{{ $$ip := .ImportPath }}{{ range .Imports}}{{ $$ip }}: {{ println . }}{{end}}' $(PKG) | grep -E ' (path|log)$$' | grep -Ev 'util/(log|stop): log'
```

Comment:
My question is why add the colon. Between the leading space and trailing $, this always matches exactly one import path.